manager/printermanager.go

```diff
@@ -225,7 +225,7 @@ func (pm *PrinterManager) applyDiff(diff *lib.PrinterDiff, ch chan<- lib.Printer
 			log.InfoPrinterf(diff.Printer.Name+" "+diff.Printer.GCPID, "Registered in the cloud")
 
 			if pm.gcp.CanShare() {
-				if err := pm.gcp.Share(diff.Printer.GCPID, pm.shareScope); err != nil {
+				if err := pm.gcp.Share(diff.Printer.GCPID, pm.shareScope, "USER", true); err != nil {
```

Comment:
Please explain why `skip_notification` should be true. Have you tested it?

I worry that the sharee wouldn't notice a new printer, making it harder to Go Find It.